### DISCUSSION TITLE
Add month in Mini Graph Card to the data points?

### DISCUSSION THREAD

**Discussion by J-o-h-n-M** [kalkih/mini-graph-card#967] [Q&A]
Is there a way to add the month to the date shown in Mini Graph Card when I hover over a data point? It now just shows “10 Mon, 14:21”. I would like it to show “Apr 10 Mon, 14:21”

![image](https://github.com/kalkih/mini-graph-card/assets/8143724/f31df776-d676-47d1-828c-83ac1db03e4a)

**Comment by Malow** [comment]
I made a fork that adds month, changes the weekday to be long format, and removes the timespan and instead shows the middle point. so instead of "10 Mon, 14:31 - 10 Mon, 15:30" it will say "Monday, 10 November at 15:00". Available here if that sounds like something you'd like: https://github.com/Malow/mini-graph-card/releases
